scripts/genproto.sh

```diff
@@ -95,18 +95,34 @@ run_go_tool github.com/hexfusion/schwag -input=Documentation/dev-guide/apispec/s
 if [ "$1" != "--skip-protodoc" ]; then
   log_callout "protodoc is auto-generating grpc API reference documentation..."
 
+  # API reference
   run rm -rf Documentation/dev-guide/api_reference_v3.md
   run_go_tool go.etcd.io/protodoc --directories="api/etcdserverpb=service_message,api/mvccpb=service_message,server/lease/leasepb=service_message,api/authpb=service_message" \
-    --title="etcd API Reference" \
     --output="Documentation/dev-guide/api_reference_v3.md" \
     --message-only-from-this-file="api/etcdserverpb/rpc.proto" \
-    --disclaimer="This is a generated documentation. Please read the proto files for more." || exit 2
+    --disclaimer='''---
+title: "API reference"
+---
 
+This API reference is autogenerated from the named \`.proto\` files.
+''' || exit 2
+
+  # remove the first 3 lines of the doc as an empty --title adds '### ' to the top of the file.
+  run sed -i -e 1,3d Documentation/dev-guide/api_reference_v3.md
+
+  # API reference: concurrency
   run rm -rf Documentation/dev-guide/api_concurrency_reference_v3.md
```

Comment:
Same as above, once we know if (and where) these file are moving, then I'd like to set an env var to help simplify this section.